app/controllers/application_controller.rb

```diff
@@ -49,7 +49,7 @@ def set_locale
   end
 
   def configure_permitted_parameters
-    devise_parameter_sanitizer.permit(:sign_up, keys: [:user_type, :name, :description, :email, :phone_number, :zip, :city, :state, :city_state, :ride_zone_id])
+    devise_parameter_sanitizer.permit(:sign_up, keys: [:user_type, :name, :description, :license_plate, :drivers_license, :email, :phone_number, :zip, :city, :state, :city_state, :ride_zone_id])
```

Comment:
Copying my comment from #934:

> Should we collect their driver's license too?

That depends on the purpose. Ride-share applications collect license plate, make, and model and send those to the rider so they can easily spot the car. DL doesn't help the rider spot the car. If there's another use-case, then possibly. But if not, it's just PII that serves to add risk to the system.

Reply:
Idea had been to collect info ride zone admins could use to vet drivers, but if they choose to do that they can collect independently, or we can add in a separate PR, so agreed.